site/docs/patterns/content-status.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Content Status
+layout: DetailPattern
+aliases:
+  - /salt/patterns/content-status
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-content-status--info", label: "Examples" },
+      { href: "https://www.figma.com/file/InTRZY7R7OdeY0Ye9eyzma/(WIP)-Content-Status?type=design&node-id=652-3271&mode=design&t=BoYoZt69VHfzizeO-0", label: "Content Status examples", internal: true },
+    ]
+---
+
+Content Status is a pattern that provides information when content isn’t ready to be shown in a container component of an application.
+
+It remains onscreen until a situation is resolved, or an action is taken by your user, making them a great tool for communication and guidance.
```

Comment:
```suggestion
It remains onscreen until a situation is resolved, or an action is taken by your user, making them a great tool for communication and guidance.
```
This might just be me but we switch to referring to Content Status as plural (them) at the end of this sentence cc @mikearildbrown